File: problem-solving/beecrowd/1051-ir/main.c
#include <stdio.h>

int main(){
  int i=0;
  float salary, rest, tax=0;
  float SALARIES[] = {4500, 3000, 2000};
  float ALIQUOTS[] = {0.28, 0.18, 0.08};

  scanf("%f", &salary);

  for (i=0; i < 3; i++){
    rest = salary - SALARIES[i];
    if (rest > 0){
      tax = tax + (rest * ALIQUOTS[i]);
      salary = salary - rest;
    }
  }
  if (tax == 0){
    printf("Isento\n");
  }
  else{
    printf("R$ %.2f\n", tax);
  }
}

File: problem-solving/beecrowd/1051-ir/main.py
SALARIES = [4500, 3000, 2000]
ALIQUOTS = [0.28, 0.18, 0.08]
tax=0

salary = float(input())

for i in range(3):
    rest = salary - SALARIES[i]
    if (rest > 0):
        tax = tax + (rest * ALIQUOTS[i])
        salary = salary - rest

if (tax == 0):
    print("Isento")
else:
    print(f"R$ {tax:.2f}")
